### ISSUE TITLE
Okta Group Object Missing Custom Attributes

### ISSUE THREAD

**Issue by jlrgraham23** [okta/okta-sdk-python#279]
Group objects returned via the SDK are missing any custom attributes that are placed on them.

A similar issue is discussed in https://github.com/okta/okta-sdk-python/issues/230, but the resolution there seems to be around the creation or mangement of custom group profile attributes.

Looking at the SDK code, it looks like the the `okta/models/group_profile.py` [Group class](https://github.com/okta/okta-sdk-python/blob/master/okta/models/group_profile.py#L33) is missing a handler for any attribute other than `name` or `description`.  In `okta/models/user_profile.py` [User class](https://github.com/okta/okta-sdk-python/blob/master/okta/models/user_profile.py#L132) this is implemented in custom attribute handling block.

Adding debug statements to the SDK shows that these attributes are being returned by the Okta API.

**Comment by laura-rodriguez** [comment]
Hi @jlrgraham23,

Thanks for your issue. Someone from our team will take a look soon.

cc @serhiibuniak-okta

**Comment by serhiibuniak-okta** [comment]
@jlrgraham23 could you please, add some usage samples or request/response samples? If we are talking about #230, then custom attributes appears within GroupSchema object: https://github.com/okta/okta-sdk-python#manage-group-schema-custom-atributes.
Maybe, you've got some custom attributes in response for GroupProfile object, then it would be good if you can share how you've created custom attributes and share raw request/response (curl/postman, etc.)

**Comment by jlrgraham23** [comment]
> @jlrgraham23 could you please, add some usage samples or request/response samples? If we are talking about #230, then custom attributes appears within GroupSchema object: https://github.com/okta/okta-sdk-python#manage-group-schema-custom-atributes. Maybe, you've got some custom attributes in response for GroupProfile object, then it would be good if you can share how you've created custom attributes and share raw request/response (curl/postman, etc.)

@serhiibuniak-okta right, the group schema can be managed there, but the values placed into said custom attributes are not included in the resulting objects when querying Okta.

A `curl` request, the desired values are the `groupOwners` element of the `profile` portion of the response, where `groupOwners` is a custom attribute I defined in the Okta profile editor.

```
bash-5.0# curl -s -H "Authorization: SSWS $OKTA_CLIENT_TOKEN" https://dev-000000.oktapreview.com/api/v1/groups/00g15iu03heJX6Qb90h8 | jq -r .
{
  "id": "00g15iu03heJX6Qb90h8",
  "created": "2022-02-16T15:02:51.000Z",
  "lastUpdated": "2022-02-16T15:03:05.000Z",
  "lastMembershipUpdated": "2022-02-16T15:02:51.000Z",
  "objectClass": [
    "okta:user_group"
  ],
  "type": "OKTA_GROUP",
  "profile": {
    "name": "okta-bug-sample",
    "description": null,
    "groupOwners": [
      "somevalue"
    ]
  },
  "_links": <removed for brevity>,
}
```

The same group as returned and processed by the SDK, the value of `groupOwners` is missing from the `profile` section:

```
{
 '_links': <removed for brevity>,
 'created': '2022-02-16T15:02:51.000Z',
 'id': '00g15iu03heJX6Qb90h8',
 'lastMembershipUpdated': '2022-02-16T15:02:51.000Z',
 'lastUpdated': '2022-02-16T15:03:05.000Z',
 'objectClass': ['okta:user_group'],
 'profile': {'name': 'okta-bug-sample'},
 'type': <GroupType.OKTA_GROUP: 'OKTA_GROUP'>
}
```

The above response is from a script like:

```
async def group_details():
    group, resp, err = await okta_client.get_group("00g15iu03heJX6Qb90h8")
    return group

group = loop.run_until_complete(group_details())
pprint(group.as_dict())
```

Digging a bit at the code, it looks like the `Group` class is missing a handler to process any attribute in the profile value other than `name` or `description`, I think the fix is something like: https://github.com/okta/okta-sdk-python/pull/280

Please let me know if more details are needed.

**Comment by jlrgraham23** [comment]
> Regarding #280 - it missing few points to be fully working, but I can fix it. If you want your PR merged, then you need to sign the CLA https://developer.okta.com/cla/ Otherwise I'll fix this issue without your PR.
> 
> Please, let me know what works for you.

Thank you @serhiibuniak-okta , I'm not particular about my PR getting merged, likely easier if you fix directly than running a CLA by legal for me to sign!   Thank you!

**Comment by serhiibuniak-okta** [comment]
@jlrgraham23 Created A PR, waiting for a review.

**Comment by jlrgraham23** [comment]
Good morning @serhiibuniak-okta, thank you for the work here.  Any thoughts on when this might get merged and released?  We're looking to develop some additional API integrations with Okta and this is a blocker for us.

**Comment by jlrgraham23** [comment]
Good morning, just wanted to check in and see if there was any progress on getting this reviewed and merged.  Anything I can do to assist?

**Comment by steveAG** [comment]
+1, any updates on this?
It's a blocker for one of our projects as well.

**Comment by bretterer** [comment]
This has been released in v2.6.0
